src/intrinsic/simd.rs

```diff
@@ -432,30 +431,47 @@ pub fn generic_simd_intrinsic<'a, 'gcc, 'tcx>(bx: &mut Builder<'a, 'gcc, 'tcx>,
 
         let vec_ty = bx.cx.type_vector(elem_ty, in_len);
 
-        let (intr_name, fn_ty) =
+        let (intr_name, num_args) =
```

Comment:
Can you please add a comment to explain why `num_args` is useful here?
I don't find it obvious: is it because `args` contain more arguments for LLVM in some cases?

Reply:
I had this because I was trying to be defensive about argument handling and only pass the number of arguments each function was expecting.  On further review, `args` should always be correct coming from rustc (AFAICT validation of these intrinsics' signatures happens before we see it), so it's probably safe to refactor `num_args` out.

Reply:
Fixed in 9691257.